web_common/web_common/styles/main.scss

```diff
@@ -425,3 +425,37 @@ pre {
   word-wrap: normal;
   white-space: pre;
 }
+
+//Progress bar css
+
+div {
+  display: inline-block;
+  margin: 0;
```

Comment:
We can't override the style for all `div`s because that breaks, for example, the line-wrapping on the Pools page:
![Screen Shot 2021-02-26 at 1 11 42 AM](https://user-images.githubusercontent.com/106194/109240595-a54aa400-77cf-11eb-92fc-f16e8ccd9d4b.png)

Should this maybe be `div.bar`?

Reply:
(for reference, the pools page should look like this:
![Screen Shot 2021-02-26 at 1 14 03 AM](https://user-images.githubusercontent.com/106194/109240745-f3f83e00-77cf-11eb-9ff8-355e39da5865.png)

)